### PR TITLE
feat: integration test

### PR DESCRIPTION
**Pull request by cap-ang**
#### List of Changes

- integration test migration from  debt-position mono repository: https://github.com/pagopa/pagopa-debt-position
- integration test GHA added

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Review by jacopocarlini — CHANGES_REQUESTED** [review]
can you use the [new](https://github.com/pagopa/pagopa-gpd-payments/pull/41/files) approach for integration tests?

File: integration-test/src/config/.env.local [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Host addresses
+host_header=api.dev.platform.pagopa.it
+api_config_host=https://api.dev.platform.pagopa.it/apiconfig/auth/api/v1
+gpd_host=https://localhost:8080/gpd/api/v1 # TODO to be changed, only a placeholder
+reporting_analysis_host=https://localhost:8080/gpd-reporting/api/v1 # TODO to be changed, only a placeholder
```
Comment (FedericoRuzzier) [inline integration-test/src/config/.env.local]: Remove TODOs

File: integration-test/README.MD [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+```
+export API_CONFIG_SUBSCRIPTION_KEY=##SUBSCRIPTION_KEY## \
+export GPD_SUBSCRIPTION_KEY=##SUBSCRIPTION_KEY## \
+export PAYMENTS_REST_SUBSCRIPTION_KEY=##SUBSCRIPTION_KEY## \
+export PAYMENTS_SOAP_SUBSCRIPTION_KEY=##SUBSCRIPTION_KEY## \
+export REPORTING_SUBSCRIPTION_KEY=##SUBSCRIPTION_KEY## \
+export REPORTING_BATCH_CONNECTION_STRING=##QUEUE_CONNECTION_STRING## \    
+yarn install && \
+yarn test:${{ parameters.ENV }}
+```
```
Comment (FedericoRuzzier) [inline integration-test/README.MD:11]: There is only one key in the implementation, update README

**Review by FedericoRuzzier — APPROVED** [review]
LGTM 👍 , take a look at comments before merging

**Pull request merged**